tests/product/prestoadmin_installer.py

```diff
@@ -51,14 +52,11 @@ def install(self, cluster=None, dist_dir=None):
         if not dist_dir:
             dist_dir = self._build_dist_if_necessary(cluster)
         self._copy_dist_to_host(cluster, dist_dir, cluster.master)
-        cluster.copy_to_host(
-            LOCAL_RESOURCES_DIR + "/install-admin.sh", cluster.master)
-        cluster.exec_cmd_on_host(
-            cluster.master,
-            'chmod +x ' + cluster.mount_dir + "/install-admin.sh"
-        )
-        cluster.exec_cmd_on_host(
-            cluster.master, cluster.mount_dir + "/install-admin.sh")
+        with open(LOCAL_RESOURCES_DIR + "/install-admin.sh", 'r') as file_obj:
+            script = ''.join(file_obj.readlines())
+
+        script = script.format(cluster.mount_dir)
```

Comment:
Prefer dict-based formatting here where the format string and args are in 2 totally different places.